content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -351,6 +351,16 @@ kubectl [flags]
 <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to false, turns off extra HTTP headers detailing invoked kubectl command (Kubernetes version v1.22 or later)</td>
 </tr>
 
+
+
+<tr>
+<td colspan="2">KUBECTL_EXPLAIN_OPENAPIV3</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Toggles whether calls to `kubectl explain` use the new OpenAPIv3 data source available. OpenAPIV3 is enabled by default since Kubernetes 1.24.
+</td>
+</tr>
```

Comment:
Will this change be overwritten by future automatic updates to this page? I'm not sure.

Reply:
yes, this will be overwritten when the kubectl reference docs are generated.
I'm not exactly sure but it may be better to place this somewhere in 
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

Reply:
I'm ok with having this merge now so we can have the change in the history/git log
/approve

Reply:
@reylejano I'd be happy to make the correct change to the generator of this page, but I am having trouble finding it. Could you point to the source for the "envvars" content of this page?

I am basing this work off of prior PR #28692 in which @sftim raised the same concern but the documentation merged without an explicit answer to whether this page is generated or not